test-case/check-pause-release-suspend-resume.sh

```diff
@@ -75,21 +72,21 @@ OPT_NAME['l']='loop'         OPT_DESC['l']='loop count'
 OPT_HAS_ARG['l']=1             OPT_VAL['l']=5
 
 OPT_NAME['i']='sleep-period' OPT_DESC['i']='sleep period of aplay, unit is ms'
-OPT_HAS_ARG['i']=1             OPT_VAL['i']='100'
+OPT_HAS_ARG['i']=1             OPT_VAL['i']='500'
 
 OPT_NAME['s']='sof-logger'   OPT_DESC['s']="Open sof-logger trace the data will store at $LOG_ROOT"
 OPT_HAS_ARG['s']=0             OPT_VAL['s']=1
 
+OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg path, default is environment variable: TPLG'
+OPT_HAS_ARG['t']=1         OPT_VAL['t']="$TPLG"
+
```

Comment:
First, add `-T` option:
```suggestion

 OPT_NAME['T']='type'    OPT_DESC['T']="suspend/resume type from /sys/power/mem_sleep" 
 OPT_HAS_ARG['T']=1         OPT_VAL['T']="" 
```